### ISSUE TITLE
Th element with valid colspan is missing scope of colgroup or ID value.

### ISSUE THREAD

**Issue by Reem-yahoo** [Organization-yahoo/org-repo#1]
**Rule description**  
One or more th elements, without an ARIA-assigned role, not intentionally hidden in the DOM and available to assistive technologies, with a colspan attribute set to a value of '0'; or '2' or more, does not have a scope attribute set to a text value of 'colgroup'; or an id attribute set to a non-null value.

**Tool name**  
Access Engine

**Severity**  
Critical

**WCAG Success criteria**  
[1.3.1 Info and Relationships - Level A](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships)

**Why it matters**  
Without an appropriate scope attribute on table header cells, header and data cells will not be properly associated. When data cells are not properly associated with header cells, users of assistive technology may not be able to reliably identify headers and navigate data tables. 

**What to do**  
Set the <code>scope</code> attribute of each cell with a colspan of 0 or 2 or more to scope="colgroup" or ensure the table header cell is associated to data cells via a valid ID attribute.

**Task in the Level Access Platform**  
https://flightpath.hub.dev.dev.levelaccess.io/my-tasks/697b8ec98542250bb269ea57/6a5680accc48a9f7e7c7031d

**Comment by Reem-yahoo** [comment]
Task pushed to GitHub Issues by Reem Aldroubi- new dev from Level Access Platform.

Posted by Reem Aldroubi- new dev via Level Access

**Comment by Reem-yahoo** [comment]
Task pushed to GitHub Issues by Reem Aldroubi- new dev from Level Access Platform.

Posted by Reem Aldroubi- new dev via Level Access

**Comment by Reem-yahoo** [comment]
Task pushed to GitHub Issues by Reem Aldroubi- new dev from Level Access Platform.

Posted by Reem Aldroubi- new dev via Level Access

**Comment by Reem-yahoo** [comment]
Task pushed to GitHub Issues by Reem Aldroubi- new dev from Level Access Platform.

Posted by Reem Aldroubi- new dev via Level Access

**Comment by Reem-yahoo** [comment]
<a href="https://flightpath.hub.dev.dev.levelaccess.io/tasks/6a5680accc48a9f7e7c7031d/attachments/6a5680accc48a9f7e7c7031b/download" target="_blank" rel="noopener noreferrer">Linked Findings.csv</a>

**Comment by Reem-yahoo** [comment]
Task pushed to GitHub Issues by Reem Aldroubi- new dev from Level Access Platform.

Posted by Reem Aldroubi- new dev via Level Access
